### ISSUE TITLE
Opaque device Near clip plane setting defaults to 0.1 at run time

### ISSUE THREAD

**Issue by raedbenz** [Microsoft/MixedRealityToolkit-Unity#1337]
## Overview
Opaque device Near clip plane setting defaults to 0.1 at inspector and run time.

## Expected Behavior
Default value should be 0.3 in inspector and run time.

## Actual Behavior
CameraCache.Main.nearClipPlane does not update appropriately.

## Steps to reproduce
Select MixedRealityCameraParent --> MixedRealityCamera, the value is 0.1.
Open script in VS and the value is 0.3.
At run time print CameraCache.Main.nearClipPlane and still says 0.1.

## Unity Editor Version
MRTP3 & MRTP4

## Mixed Reality Toolkit Release Version
842c4a1

**Comment by keveleigh** [comment]
0.1 should be fine for immersive devices. Is it not?

If anything, the bug here is that the script needs to be updated to default to 0.1 instead of 0.3. The prefabs have already been updated.

**Comment by StephenHodgson** [comment]
Yeah Kev definitely a bug where it's probably not updating the CameraCache value for some reason.

**Comment by keveleigh** [comment]
I think there might be some confusion here based on prefab values vs script defaults.

Previously, the default on both was set to 0.3. The prefabs were updated recently, but the scripts weren't.

Changes made in code, but overwritten in the Inspector on a prefab, do not change the behavior. This would only change the default if the script was added to a new GameObject.

I just validated that the CameraCache is correctly getting updated to whichever value is in the Inspector.
